src/content/blog/covid-19-navadmins.md

```diff
@@ -16,6 +16,8 @@ But don't take our word for it, the actual instruction is also included!
 
 ***Note: Since this is an on-going pandemic, this post will be updated as new guidance is put out. Check back to stay updated!***
 
+If you are experiencing any difficulty during the COVID-19 crisis, please reach out to one of the resources provided by the Navy. Those resources can be found [here](https://www.thereserveforce.com/blog/navy-resources/).
```

Comment:
You should change the URL you provided to a relative URL. Instead of `[here](https://www.thereserveforce.com/blog/navy-resources/)` it should be `[here](/blog/navy-resources/)`.

Here's why. If we were to look at this link on the staging website or locally on our computer, it will link to the production version of your new blog post, not the version in staging or locally on our computer.